bin/codechecker-log

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+Entry point for the build logger command.
+"""
+
+import imp
+import os
+
+THIS_PATH = os.path.dirname(os.path.abspath(__file__))
+CC = os.path.join(THIS_PATH, "CodeChecker")
+
+# Load CodeChecker from the current folder (the wrapper script (without .py))
+CodeChecker = imp.load_source('CodeChecker', CC)
```

Comment:
No. The folder where the environment bootstrapper resides is not in the `PYTHONPATH`. Also it's not a real Python package either, and importing from relative paths outside a package is an error, and inside a package is still discouraged.

See, for example, [this question on SO](http://stackoverflow.com/a/7506029). We couldn't use `from ..bin` because we can't make assumptions on where the CC scripts that go into the users' `PATH` are installed.